pkg/collector/corechecks/oracle-dbm/custom_queries.go

```diff
@@ -120,27 +128,34 @@ func (c *Check) CustomQueries() error {
 					if v_str, ok := v.(string); ok {
 						metricRow.value, err = strconv.ParseFloat(v_str, 64)
 						if err != nil {
-							logTypeError(metricPrefix, "number", metricRow.name, v, q.Query)
+							logTypeError(metricPrefix, "number", metricRow.name, v, q.Query, err)
```

Comment:
Why only log this typing error but not add to `allErrors`?

Reply:
Fixed now, thx.